### ISSUE TITLE
[BUG] Empty BGCHANGES are replaced by .gif originally set for CDTITLE

### ISSUE THREAD

**Issue by DarkBahamut162** [TeamRizu/OutFox#510]
### Is there an existing issue for this?

- [X] I have searched the existing issues and none matched.

### Operating System

Windows 10 Home 64-bit (21H2)

### CPU

Intel Core i7 2600 @ 3.40GHz

### GPU

8192MB ATI Radeon RX 580 Series

### Storage

2x 1863GB Western Digital WDC WD2003FZEX-00SRLA0 ATA Device

### Game Version

4.14.0

### Game Mode

dance

### Theme

any...

### Describe the problem

The following has been prepped hours before the fallout within #508 caused by me:

Guys, you are giving me headaches (and anxiety)... I don't like you fixing stuff in your "HotFix Updates" while introducing bugs with them... (reason stated at second to last sentence)

*0.4.14.1's Changelog:
`Fixed bug with BGA missing when simfile author didn't add the video file to the .sm/.ssc as to the standard specification`

Uhmmm... Maybe don't do that? Please? Either that or differentiate what is an actual video file or if it is already being used/set to something else...

Logs:
`00:27.622: FFMPEG Decoder: Movie /Songs/pack/song/dark.gif has handler 'Animated Image', type 'GIF/APNG'`
`00:31.753: Loading screen: "ScreenGameplay"`
`00:31.830: SM: File has no BGA set by default, found dark.gif, Setting it as BGA'`

GIFs aren't videos... At least the one I've set as a CDTITLE isn't...
Now it changes itself to that once the song starts (reaches Beat 0)
I noticed this while theming...

The last time one hotfix after the other happened, it was already bad enough. Please don't rush things...
*Also: The Common splash file still says 0.4.14.0 instead of 0.4.14.1

### Describe what should happen

I don't care anymore...
You can close this issue immediately as I'll retroactively change said issuing .gif CDTITLE to a new .png one

### Relevant Log output

The following has been added the hour after the fallout within #508 caused by me:

I'm sorry that certain wordings I've written down at the certain issue that may have caused this very issue somehow came over the wrong way and I would like to clarify:

The "not sure what happened between 4.13 and 4.14" line was because NOT of me "not" knowing about the changelog as I have read them prior to downloading the new version. It wasn't even a SSC issue since the issue I've commented on was about a .sm file (unless said parser now does .ssc AND .sm, which I didn't know - hence the "SSC Parser"). It was mainly to what had been said last within the very issue by the issuer:

"It's definitely a 4.14 problem because the simfiles don't have BGChanges indicated, but in 4.13 they still loaded automatically. In 4.14, they don't, even if I specify BGChanges for them."

I'm not accepting an apology as it's clearly my fault on this rushed comment of mine from where I forgot to insert the comment there myself or establish the connection between my comment and my intention for said line.
I'll also refrain from reporting on any more minor bugs (unless they are major) up until AlphaV drops.

**Comment by Scraticus** [comment]
Hiya, 

Thanks for the report, we will adjust how the gif/gifv files are sorted out as we forgot about this one in honesty.

Further to your comments at the bottom of the report, we would rather you report nitpicks, bugs, feature requests than not, it just seemed like in your posts you were angry or upset at us, so your tone always seemed like we had done something wrong.

I can understand text is not the best way to follow emotion or intent, so I apologise for my comment there, just you did seem a little angry about the bug cropping up.

In short, we actually really appreciate bug reports and nitpicks even if they sometimes make us die a little inside because we forgot to check the one edge case that you or another player has found. At the end of the day, we cannot fix things we do not know about, and for that I will applaud your thoroughness and commitment to improving OutFox for everyone.

I'd say lets wipe the slate clean and move past this set of crossed wires.

**Comment by Jousway** [comment]
the problem is that we set gifs to be handled by ffmpeg, so it thinks that gifs are bga's so the bga check that checks for every FT_Movie thinks gifs are the bga.

**Comment by Scraticus** [comment]
This is fixed in 4.15, thanks for reporting
